<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>262</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>262</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LOL Champ Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>x64/Release/personal.ico</normaloff>x64/Release/personal.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="list_primary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="list_secondary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addPrimary">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_removePrimary">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addSecondary">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_removeSecondary">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_byChampion">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>By Champion</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_byCategory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>By Category</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_apply">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_restore">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>262</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileSettings"/>
    <addaction name="separator"/>
    <addaction name="fileExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTroubleshooting"/>
    <addaction name="actionAbout_Champion_Manager"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="fileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTroubleshooting">
   <property name="text">
    <string>Troubleshooting</string>
   </property>
  </action>
  <action name="actionAbout_Champion_Manager">
   <property name="text">
    <string>About Champion Manager</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
